### ISSUE TITLE
Shape mismatch during sanity check

### ISSUE THREAD

**Issue by Akhilesh64** [shabie/docformer#40]
Can the target size be changed. Currently when I try to change it throws `mat1 and mat2 shapes cannot be multiplied (3072x768 and 192x128)` . I tried with target size (1000, 768). Is something hardcoded in the code which prevents arbitrary shapes.

**Comment by uakarsh** [comment]
Yep, it is a bit hardcoded. You need to see the `ResNet Feature Extractor` in `modeling.py` for the changes. Rest all as far as I know, would be handled.
